tests/frameworks/layers/l2l4_network_creation_test.py

```diff
@@ -625,7 +795,8 @@ def testTwoColumnsL4L2DataFlow(self):
 
     self.assertEqual(
       len(net.regions.keys()), 4 * 2,
-      "Incorrect number of regions"
+      "Incorrect number of regions, expected {} but had {}".format(8*2,
+      len(net.regions.keys()))
```

Comment:
indent since you are wrapping in the middle of `format()`, not `assertEqual()` like the previous line